azureadps-2.0/AzureAD/Get-AzureADMSAuthorizationPolicy.md

```diff
@@ -0,0 +1,45 @@
+---
+external help file: Microsoft.Open.MS.GraphV10.PowerShell.dll-Help.xml
+Module Name: AzureAD
+online version:
+schema: 2.0.0
+---
+
+# Get-AzureADMSAuthorizationPolicy
+
+## SYNOPSIS
+Gets an authorization policy.
```

Comment:
When I try to find a description for what an authorization policy is, I don't get any information/definition from our Azure AD IT docs. Instead, the first hit is in the Graph docs and some blogs that don't define what it is. Here's what a Graph resource topic says about it: "Represents a policy that can control Azure Active Directory authorization settings". 

Please update this description to say "Gets an authorization policy, which represents a policy that can control Azure Active Directory authorization settings." to give more context